www/index.html

```diff
@@ -417,7 +417,8 @@ <h3>Other Collaborators</h3>
                     <li>Jack Wong - Software engineering student at University of Auckland</li>
                 </ul>
                 <h3>Acknowledgements</h3>
-                <p>Webgazer project is developed based on the research that is done in Brown University. The work of the calibration example file was developed in the context of a course project topic to improve the feedback of WebGazer. It was proposed by Dr. Gerald Weber and his team Dr. Clemens Zeidler and Kai-Cheung Leung.</p>
+                <p>Webgazer project is developed based on the research that is done in Brown University. The work of the calibration example file was developed in the context of a course project topic to improve the feedback of WebGazer. It was proposed by Dr. Gerald Weber and his team Dr. Clemens Zeidler and Kai-Cheung Leung. The cause of index html put the script link in the head instead of body. Acknowledge to This research is supported by NSF grants IIS-1464061, IIS-1552663, and the Brown University Salomon Award.
```

Comment:
Is this sentence supposed to be here: "...The cause of index html put the script link in the head instead of body. Acknowledge to..."